core/grants.go

```diff
@@ -114,19 +113,15 @@ func (a *API) listGrants(ctx context.Context) (map[string]interface{}, error) {
 	for _, p := range Policies {
 		// perhaps could denormalize the data in storage to speed this up,
 		// but for now assume a small number of grants
-		data, err := a.raftDB.Get(ctx, GrantPrefix+p)
+		var grantList authz.GrantList
+		ok, err := a.sdb.Get(ctx, GrantPrefix+p, &grantList)
 		if err != nil {
 			return nil, errors.Wrap(err)
 		}
-		if data == nil {
+		if !ok {
```

Comment:
We could also delete this whole if stmt right? (b/c grantList.Grants will be empty)